fedimint-eventlog/src/lib.rs

```diff
@@ -390,14 +390,17 @@ pub async fn run_event_log_ordering_task(
         // This thread is the only thread deleting already existing element of unordered
         // log and inserting new elements into ordered log, so it should never
         // fail to commit.
```

Comment:
Would be good to add a comment explaining fjall changes.

Reply:
done